cfgtenants.go

```diff
@@ -0,0 +1,39 @@
+package veracity
+
+import (
+	"fmt"
+	"strings"
+)
+
+// NormTenantIdentity ensures a string is prefixed with 'tenant/'
+// Note the expected input is a uuid string or a tenant/uuid string
+func NormTenantIdentity(tenant string) string {
```

Comment:
minor: I am not sure i like the word Norm, i understand we want to be robust incase we decide we hate the tenant/uuid format, but i think thats pretty much baked in, maybe call this something closer to `HasTenantPrefix`.

really minor, it is readable as it is.

Reply:
Yeah, or Ensure, since its doing the transformation if its missing

Reply:
Or just Normalize tbh

Reply:
I have a follow up PR to do to add some tests for the release. I'll pick this up there